src/github.com/getlantern/flashlight/server/server.go

```diff
@@ -240,7 +241,7 @@ func determineInternalIP() (string, error) {
 }
 
 func onBytesGiven(destAddr string, req *http.Request, bytes int64) {
-	_, port, _ := net.SplitHostPort(destAddr)
+	host, port, _ := net.SplitHostPort(destAddr)
```

Comment:
I notice we don't check for a non-nil error here, but we do in other places where we call SplitHostPort. I haven't traced the call graph, but is that because destAddr is guaranteed to be of a valid format when we arrive here?